test/services/SubmissionServiceSpec.scala

```diff
@@ -92,8 +94,14 @@ class SubmissionServiceSpec extends SpecBase with MockitoSugar with BeforeAndAft
       }
     }
 
+    "calculate the total rate as £56" when {
+      "given a start date of 1/1/2020 (the latest 2019 claiming date)" in new Setup {
+        serviceUnderTest.calculate2019FlatRate(YEAR_2020_START_DATE) shouldBe 56
+      }
+    }
+
     "calculate the total rate as £212" when {
-      "given a start date of 6/4/2019 (the first day of the 2019 tax year)" in new Setup {
+      "given a start date of 6/4/2019 (the start of the 2019 tax year)" in new Setup {
         serviceUnderTest.calculate2019FlatRate(TAX_YEAR_2019_START_DATE) shouldBe 212
       }
       "given a start date of 6/4/1999 (the first day of the 1999 tax year)" in new Setup {
```

Comment:
This test is failing, I just think we need to assert it as £56 rather than the £212 (due to the earliest date change being 1/1/2020 instead of 6/4/2019)